.travis.yml

```diff
@@ -72,11 +72,8 @@ script:
 # Deploy debian packages to bintray
 deploy:
     provider: script
-    script: beaver bintray upload -N pkg/deb/*.deb
+    script: beaver bintray upload -N deb/*.deb
     skip_cleanup: true
     on:
         tags: true
-        condition: >
-            "$TRAVIS_OS_NAME" = "linux" -a
-            "$CC" = "gcc"
-
+        condition: $TRAVIS_OS_NAME = 'linux' && $CC = 'gcc'
```

Comment:
Tried setting the the options with `bash -c "set -eux"` 
However  AFAICS it doesn't affect  the verbosity output compared to earlier builds `.travis`.
However we might set the options in v5.x.x which adds a separate bash script. 
Did experience though that the build errored due to the extensive logging after trying `set -v`